src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        // Читання даних із файлу
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] split = line.split(",");
+                String operation = split[0];
+                int number = Integer.parseInt(split[1]);
```

Comment:
Consider adding error handling for `Integer.parseInt(split[1])` to catch potential `NumberFormatException` if the input data is not a valid integer. This will make your code more robust against malformed input data.